From: Ubuntu Core Developers <ubuntu-devel-discuss@lists.ubuntu.com>
Date: Thu, 14 Feb 2019 18:59:14 -0800
Subject: bdist-wininst-notfound

# DP: suggest installation of the pythonX.Y-dev package, if bdist_wininst
# DP: cannot find the wininst-* files.
# DP: suggest installation of the pythonX.Y-dev package, if bdist_wininst
# DP: cannot find the wininst-* files.
---
 Lib/distutils/command/bdist_wininst.py | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/Lib/distutils/command/bdist_wininst.py b/Lib/distutils/command/bdist_wininst.py
index 3a61688..21b83ee 100644
--- a/Lib/distutils/command/bdist_wininst.py
+++ b/Lib/distutils/command/bdist_wininst.py
@@ -360,6 +360,10 @@ class bdist_wininst(Command):
 
         filename = os.path.join(directory, "wininst-%s%s.exe" % (bv, sfix))
         f = open(filename, "rb")
+        try:
+            f = open(filename, "rb")
+        except IOError as e:
+            raise DistutilsFileError(str(e) + ', %s not included in the Debian packages.' % filename)
         try:
             return f.read()
         finally:
